src/ir/objc.rs

```diff
@@ -0,0 +1,78 @@
+//! Objective C types
+
+use clang;
+use clang_sys::CXChildVisit_Continue;
+use clang_sys::CXCursor_ObjCInstanceMethodDecl;
+// use clang_sys::CXCursor_ObjCSuperClassRef;
+use super::context::BindgenContext;
+
+/// Objective C interface as used in TypeKind
+///
+/// Also protocols are parsed as this type
+#[derive(Debug)]
+pub struct ObjCInterface {
+    /// The name
+    /// like, NSObject
+    pub name: String,
+
+    /// List of the methods defined in this interfae
+    pub methods: Vec<ObjCInstanceMethod>,
```

Comment:
instead of making these public, could/should we make them private and add accessors?